examples/prometheus/02-prometheus-configmap.yaml

```diff
@@ -45,7 +45,7 @@ data:
       tls_config:
         ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
         # If your node certificates are self-signed or use a different CA to the
-        # master CA, then disable certificate verification below. Note that
+        # main CA, then disable certificate verification below. Note that
```

Comment:
need to rephrase this

Reply:
This is a straight copy of an upstream thing, I think. I'll just revert.